src/PattyController.js

```diff
@@ -21,14 +19,16 @@ class PattyController {
    * @param {PattyHome} home
    * @param {PattyOptions} options
    * @param {ServerLogger} logger
+   * @param {function} exitHandler
    */
-  constructor(home, options, logger) {
+  constructor(home, options, logger, exitHandler) {
```

Comment:
the exitHandler function will be passed by the PattyServer and will close the HTTP server when a kill is requested